CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - It is now possible to generate a new BIB database from the citations in an OpenOffice/LibreOffice document
 - The arXiv fetcher now also supports free-text search queries
 - [#1345](https://github.com/JabRef/jabref/issues/1345) Cleanup ISSN
+- [#898](https://github.com/JabRef/jabref/issues/898) Rewrite bibtexml importer with JAXB parser
```

Comment:
Make a less technical description in the changelog and rather explain what has changed for the user.